perf/istio-install/setup_prometheus.sh

```diff
@@ -15,16 +16,25 @@ function install_prometheus() {
   helm fetch stable/prometheus-operator --untar --untardir "${PROMETHEUS_INSTALL}"
 
   kubectl create ns istio-prometheus || true
-  PROM_OP="$DIRNAME/prom-operator.yaml"
+  PROM_OP="${DIRNAME}prom-operator.yaml"
+  PROM_CRDS="${DIRNAME}prom-op-crds.yaml"
+  cd "${PROMETHEUS_INSTALL}/prometheus-operator"
+  ls ./templates/prometheus-operator/crd*.yaml | \
```

Comment:
Maybe 
```
kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/alertmanager.crd.yaml
kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/prometheus.crd.yaml
kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/prometheusrule.crd.yaml
kubectl apply -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/example/prometheus-operator-crd/servicemonitor.crd.yaml
```

From https://github.com/helm/charts/tree/master/stable/prometheus-operator#helm-fails-to-create-crds

Reply:
Also ` kubectl wait --for=condition=Established -f "${DIRNAME}/${release}/istio/templates/crds.yaml"`